java/mvnd/Portfile

```diff
@@ -31,12 +30,14 @@ long_description mvnd aims at providing faster Maven builds using techniques kno
                 This applies not only to the code coming from Maven plugins and Maven Core,\
                 but also to all code coming from the JDK itself.
 
-github.tarball_from releases
-distname        ${name}-${version}-darwin-amd64
+homepage        https://github.com/apache/maven-mvnd
 
-checksums       rmd160  a20216cb788f0182c4af3249a4a2a8af4ca1a50a \
-                sha256  7a0de6107b9e19290ccc6853ea93a087e5fe1558f64eab66a9e803a03afedc3c \
-                size    26546783
+master_sites    https://downloads.apache.org/maven/mvnd/${version}/
+distname        maven-mvnd-${version}-darwin-amd64
```

Comment:
should one not set "supported_arch" to the supported versions? Based on the filename it will likely not support i386 for example?

Reply:
can you comment on this? If I'm mistaken about it then this should be ready to merge

Reply:
I've added `supported_archs x86_64` now.

Reply:
okay, but is that correct? I am just asking the question, not saying that it should be `x86_64` per se. For example, it appears that the port does build on `arm64` (https://ports.macports.org/port/mvnd/details/). Again, I don't know as I haven't investigated what upstream supports

Reply:
I think that is correct. This port is using a pre-built binary.